apps/src/codebridge/MiniAppPreview/NeighborhoodPreview.tsx

```diff
@@ -19,7 +19,11 @@ import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
 const NeighborhoodPreview: React.FunctionComponent = () => {
   const levelProperties = useAppSelector(state => state.lab.levelProperties);
   const {source, config} = useCodebridgeContext();
-  const serializedMaze = findFile(source, MAZE_FILE_NAME, DEFAULT_FOLDER_ID);
+  const serializedMaze = findFile(
```

Comment:
Assign to stringified maze so that the `useEffect` below is only triggered when the maze contents change. Previously, the `useEffect` was triggered at the end of each run since `serialized_maze.txt` is now part of the project source. Even if the contents hadn't changed, the `source` object was updated.

Reply:
Thanks @molly-moen for helping me dig into this bug which was causing the `Done` signal to be processed immediately so that other maze commands were not being processed.